src/bootstrap/src/core/builder.rs

```diff
@@ -714,11 +708,8 @@ impl Kind {
             Kind::Setup => "setup",
             Kind::Suggest => "suggest",
             Kind::Vendor => "vendor",
-<<<<<<< HEAD
             Kind::Sign => "sign",
```

Comment:
would keep this at the end since it's specific to Ferrocene, maybe even comment it as such